libbpf-tools/profile.c

```diff
@@ -584,7 +584,7 @@ int main(int argc, char **argv)
 	libbpf_set_print(libbpf_print_fn);
 
 	nr_cpus = libbpf_num_possible_cpus();
-	if (nr_cpus < 0) {
+	if (nr_cpus <= 0) {
 		printf("failed to get # of possible cpus: '%s'!\n",
 		       strerror(-nr_cpus));
```

Comment:
I agree with you that a result of 0 from libbpf_num_possible_cpus can be treated as a failure case. 

However, on line 589, strerror(0) will print 'Success'. Please check this. Thank you.

Reply:
I agree with you. So I changed the code. I hope you to review it again.
Thanks for the review.

Reply:
How about using just one if statement for the failure case?

Reply:
I changed the code to use one if statement for the failure case.